scripts/zones/Pashhow_Marshlands/DefaultActions.lua

```diff
@@ -3,4 +3,5 @@ local ID = zones[xi.zone.PASHHOW_MARSHLANDS]
 return {
     ['Odyssean_Passage'] = { messageSpecial = ID.text.NOTHING_HAPPENS },
     ['Outpost_Gate']     = { messageSpecial = ID.text.GATE_IS_LOCKED },
+    ['QM1']              = { messageSpecial = ID.text.NOTHING_OUT_OF_ORDINARY },
```

Comment:
`QM1`?
qm1

`INSERT INTO `npc_list` VALUES (17146515,'qm1','???',0,-105.147,24.999,-255.986,1,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'WOTG',0);
`

Reply:
That's why they pay you the big bucks 🫠
Fixed!

Reply:
If I got paid this wouldnt be a hobby